www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -23,224 +23,282 @@ description: How to manage platforms and Cordova CLI versions.
+* **`cordova platform add android@7.1.4`**
+
+  Retrieves the `cordova-android` platform version `7.1.4` from npm, adds it to the project and updates the `package.json` file.
+
+* **`cordova platform add https://github.com/apache/cordova-android.git`**
+  
+  **`cordova platform add https://github.com/apache/cordova-android`**
+  
+  **`cordova platform add github:apache/cordova-android`**
+
+  npm retrieves the `cordova-android` platform from the git repository, adds it to the project and updates the `package.json`.
+  
+* **`cordova platform add C:/path/to/android/platform`**
+
+  Retrieves the Android platform from the specified directory, adds it to the project, and updates the `package.json` file.
+
+* **`cordova platform add android --nosave`**
 
-### Updating / Removing platforms
-It is also possible to update/delete from config.xml and package.json during the commands 'cordova platform update' and 'cordova platform remove' :
+  Retrieves the pinned version of `cordova-android` platform from npm, adds it to the project, but does not add it to the `package.json` file.
+
+### Updating or Removing Platforms
+
+It is also possible to update or delete from `config.xml` and `package.json` during the commands 'cordova platform update' and 'cordova platform remove' :
 
 ```bash
-$ cordova platform update <platform[@<version>] | directory | git_url> --save
-$ cordova platform remove <platform>
+cordova platform update <platform[@<version>] | directory | git_url> --save
+cordova platform remove <platform>
 ```
 
-Some examples :
+Some Examples:
+
+* **`cordova platform update android --save`**
+
+  In addition to updating the `cordova-android` platform to the pinned version, it updates the `package.json` file.
+
+* **`cordova platform update android@3.8.0 --save`**
+
+  In addition to updating the `cordova-android` platform to version `3.8.0` it updates the `package.json` file.
+
+* **`cordova platform update /path/to/android/platform --save`**
+
+  In addition to updating the `cordova-android` platform to version in the folder it updates the `package.json` file.
+
+* **`cordova platform remove android`**
 
-  * **'cordova platform update android --save'** => In addition to updating the android platform to the pinned version, update config.xml entry
-  * **'cordova platform update android@3.8.0 --save'** => In addition to updating the android platform to version 3.8.0, update config.xml entry
-  * **'cordova platform update /path/to/android/platform --save'** => In addition to updating the android platform to version in the folder, update config.xml entry
-  * **'cordova platform remove android'** => Removes the android platform from the project and deletes its entry from config.xml and package.json.
+  Removes the `cordova-android` platform from the project and removes from the `package.json` file.
+  
+  _Note: If the platform definiton existed in `config.xml` from a previous version of Cordova-CLI, it will also be removed from `config.xml`._
```

Comment:
```suggestion
  _Note: If the platform definiton existed in `config.xml` from a previous version of Cordova CLI, it will also be removed from `config.xml`._
```